server/lib/email.ts

```diff
@@ -125,8 +125,6 @@ const sendMessage = (
   html: string,
   options: SendMessageOptions = {},
 ) => {
-  options.bcc = options.bcc || 'emailbcc@opencollective.com';
```

Comment:
@znarf Should we keep it for staging? I still find it useful to have a place where I can get all staging emails.

Reply:
I'm not relying on it personally. We don't send much emails on staging?

Why not look at Mailgun logs instead?

Reply:
*Note: this was discussed in a call*